src/components/counter-element.ts

```diff
@@ -19,24 +19,27 @@ import { ButtonSharedStyles } from './button-shared-styles.js';
 // This is a reusable element. It is not connected to the store. You can
 // imagine that it could just as well be a third-party element that you
 // got from someone else.
-class CounterElement extends LitElement {
+@customElement('counter-element')
+export class CounterElement extends LitElement {
   @property({type: Number})
   clicks = 0;
 
   @property({type: Number})
   value = 0;
 
-  static styles = [
-    ButtonSharedStyles,
-    css`
-      span {
-        width: 20px;
-        display: inline-block;
-        text-align: center;
-        font-weight: bold;
-      }
-    `
-  ];
+  static get styles() {
```

Comment:
Why `static get style() { return []; }` instead of `static styles = [];`?

Reply:
apparently is for polymer-cli support
https://github.com/Polymer/lit-element/issues/584
https://github.com/Polymer/lit-element/pull/585